iron-meta.html

```diff
@@ -268,66 +137,64 @@
         value: {
           type: Object,
           notify: true,
-          readOnly: true
         },
 
         /**
-         * Array of all meta-data values for the given type.
+         * If true, `value` is set to the iron-meta instance itself.
          */
-        list: {
-          type: Array,
-          notify: true
+         self: {
+          type: Boolean,
+          observer: '_selfChanged'
+        },
+
+        __meta: {
+          type: Boolean,
+          computed: '__computeMeta(type, key, value)'
         }
+      },
 
+      hostAttributes: {
+        hidden: true
       },
 
-      /**
-       * Actually a factory method, not a true constructor. Only runs if
-       * someone invokes it directly (via `new Polymer.IronMeta()`);
-       *
-       * @param {{type: (string|undefined), key: (string|undefined)}=} config
-       */
-      factoryImpl: function(config) {
-        if (config) {
-          for (var n in config) {
-            switch(n) {
-              case 'type':
-              case 'key':
-                this[n] = config[n];
-                break;
-            }
-          }
+      __computeMeta: function(type, key, value) {
+        let meta = new Polymer.IronMeta({
+          type: type,
+          key: key
+        });
+
+        if (value !== undefined && value !== meta.value) {
```

Comment:
I'm going to wait on reviewing this element as setting `value`as a setter and getter seems to be acting weird with Polymer despite it being set on the element's markdown.